server.tf

```diff
@@ -0,0 +1,294 @@
+  template = "${local.server_environment}"
+
+  vars {
+    forseti_home             = "${var.forseti_home}"
+    forseti_server_conf_path = "${local.server_conf_path}"
+    storage_bucket_name      = "${local.server_bucket_name}"
+  }
+}
+
+data "template_file" "forseti_server_env" {
+  template = "${local.server_env}"
+
+  vars {
+    project_id             = "${var.project_id}"
+    cloudsql_db_name       = "${var.cloudsql_db_name}"
+    cloudsql_db_port       = "${var.cloudsql_db_port}"
+    cloudsql_region        = "${var.cloudsql_region}"
+    cloudsql_instance_name = "${google_sql_database_instance.master.name}"
+  }
+}
+
+data "template_file" "forseti_server_config" {
+  template = "${local.server_conf}"
+
+  vars {
+    root_resource_id        = "${local.root_resource_id}"
+    gsuite_admin_email      = "${var.gsuite_admin_email}"
+    forseti_email_sender    = "${var.forseti_email_sender}"
+    forseti_email_recipient = "${var.forseti_email_recipient}"
+    storage_bucket_name     = "${local.server_bucket_name}"
+    sendgrid_api_key        = "${var.sendgrid_api_key}"
+  }
+}
+
+#-------------------#
+# Activate services #
+#-------------------#
+resource "google_project_service" "activate_services" {
+  count              = "${length(local.services_list)}"
+  project            = "${var.project_id}"
+  service            = "${local.services_list[count.index]}"
+  disable_on_destroy = "false"
+}
+
+#-------------------------#
+# Forseti Service Account #
+#-------------------------#
+resource "google_service_account" "forseti_server" {
+  account_id   = "${local.server_sa_name}"
+  project      = "${var.project_id}"
+  display_name = "Forseti Server Service Account"
+}
+
+resource "google_project_iam_member" "server_roles" {
+  count   = "${length(local.server_project_roles)}"
+  role    = "${local.server_project_roles[count.index]}"
+  project = "${var.project_id}"
+  member  = "serviceAccount:${google_service_account.forseti_server.email}"
+}
+
+resource "google_organization_iam_member" "org_read" {
+  count  = "${var.org_id != "" ? length(local.server_read_roles) : 0}"
+  role   = "${local.server_read_roles[count.index]}"
+  org_id = "${var.org_id}"
+  member = "serviceAccount:${google_service_account.forseti_server.email}"
+}
+
+resource "google_organization_iam_member" "folder_read" {
+  count     = "${var.folder_id != "" ? length(local.server_read_roles) : 0}"
+  role      = "${local.server_read_roles[count.index]}"
+  folder_id = "${var.folder_id}"
+  member    = "serviceAccount:${google_service_account.forseti_server.email}"
+}
+
+resource "google_organization_iam_member" "org_write" {
+  count  = "${var.org_id != "" && var.enable_write ? length(local.server_write_roles) : 0}"
+  role   = "${local.server_write_roles[count.index]}"
+  org_id = "${var.org_id}"
+  member = "serviceAccount:${google_service_account.forseti_server.email}"
+}
+
+resource "google_organization_iam_member" "folder_write" {
```

Comment:
this should be google_folder_iam_member since the permission is applied on a folder